docs/python-bindings.md

```diff
@@ -116,3 +116,18 @@ properties:
 
 These are the same properties as those of `opengen.tcp.SolverStatus`.
 
+
+## Importing optimizer with variable name
+
+Previously we used `import rosenbrock` to import the auto-generated module.
+
+The limitation of this syntax is that it makes it difficult to change the name of the optimizer, i.e., `rosenbrock` is hard-coded.
```

Comment:
an alternative is the `import rosenbrock as my_rosenbrock`

Reply:
but `__import__` is fine as well

Reply:
`__import__` might be better if for any reason the user needs to define the name of the optimizer dynamically (e.g., if they want to create several optimizers called `rosenbrock1`, `rosenbrock2` etc.